### DISCUSSION TITLE
Add possibility to toggle power zigbee devices in webui for Sonoff ZbBridge

### DISCUSSION THREAD

**Discussion by midnight1979** [arendst/Tasmota#16405] [Ideas]
Does it possible to make the power state "clickable" in webui for power zigbee devices in Sonoff Zbbridge tasmota? 
It will be very useful feature, especially when tune the system and for quick check of power sockets for example without using console commands for them.

Just open webui and click on state of power socket to toggle the power state. 
![Screenshot_20220901-010524_Chrome.jpg](https://user-images.githubusercontent.com/18383176/187772087-d9c93176-442d-4cb0-ae51-fd6100a4d545.jpg)

**Comment by sfromis** [comment]
Well, the rudimentary Tasmota web UI is not meant as a "control panel" for daily operation of devices, more for a bit of admin/debug, with operation expected to happen via your backend dashboard setup (or automations).

**Comment by midnight1979** [comment]
> Well, the rudimentary Tasmota web UI is not meant as a "control panel" for daily operation of devices, more for a bit of admin/debug, with operation expected to happen via your backend dashboard setup (or automations).

It is doesn't very hard coding feature I hope and not really increase firmware size, but will be very helpful wile tuning the system or while debug some situations with the  backend.

**Comment by midnight1979** [comment]
While debugging the feedback state from the power socket or after pairing a new socket to the bridge and momentary check of it workability.

**Comment by s-hadinger** [comment]
I will be working on a more general way to enrich the UI with Berry. This will of course be only available on ESP32 based coordinators

**Reply by wayner9** [reply]
Did this ever happen?

**Reply by s-hadinger** [reply]
Yes see here: https://github.com/arendst/Tasmota/discussions/23653#discussioncomment-13970372

**Comment by midnight1979** [comment]
> I will be working on a more general way to enrich the UI with Berry. This will of course be only available on ESP32 based coordinators

In this case I must to plan purchase sonoff zbbridge pro?

**Reply by s-hadinger** [reply]
If you want more than what exists today yes. We have reached the limits of esp8266
